packages/atomic/src/components/facets-v1/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -110,16 +115,14 @@ export class AtomicNumericFacet
   @Prop() public field!: string;
   /**
    * The number of values to request for this facet, when there are no manual ranges.
+   * If the number of values is 0, no ranges will be displayed.
    */
   @Prop() public numberOfValues = 8;
   /**
    * Whether this facet should contain an input allowing users to set custom ranges.
+   * The depending on the field, the input can either allow decimal values or only integers.
```

Comment:
Is there a word missing between `The` and `depending`?

Reply:
```suggestion
   * Depending on the field, the input can allow either decimal or integer values.
```